tests/kernel/update_kernel.pm

```diff
@@ -121,9 +120,6 @@ sub kgraft_state {
 
 sub install_lock_kernel {
     my $version = shift;
-    if ($version eq '4.12.14-25.13.1')     { $wk_ker = 1; }
-    if ($version eq '4.12.14-197.15.1')    { $wk_ker = 2; }
```

Comment:
This will break SLE-15SP1 livepatch testing. We need this exception at least for another 3 months or so.
https://openqa.suse.de/tests/4354818#step/install_ltp/2

Reply:
As outlined in the commit log, I don't think it will.

The effect of $wk_ver == 2 was that the livepatch upgrade had been done by means of

  zypper in kernel-livepatch-4_12_14-197_15-default

instead of the regular

  zypper in -l -t patch $patches"

AFAICT, this used to be necessary because SLE15-SP1_Update_4 lacked the initial livepatch for some reason and thus, the preceeding KLP product installation wouldn't have installed any package the above 'zypper in -t patch' could then apply patches to. As we have released a couple of livepatches for this kernel in the meanwhile (available at the repo added in install_klp_product()), this problem should not persist anymore.

However, given that there aren't any KLP incidents pending atm, I can't test this claim and would be fine with reintroducing that $wk_ver == 2 logic, if wanted. Shall I?

Reply:
OK, never mind then.